### ISSUE TITLE
[robonect] documentation wrong and partially displayed corrupted

### ISSUE THREAD

**Issue by reyem** [openhab/openhab2-addons#3698]
## Expected Behavior
The documentation should be correct with examples and be formatted readable.

## Current Behavior
The documentation is wrong about:
* mower mode channel (wrong value for MANUAL)
* wrong examples for JOB channel configuration
* corrupted format on https://docs.openhab.org/addons/bindings/robonect/readme.html

## Possible Solution
Document MANUAL instead of MAN for the mower mode channel
provide proper examples for the JOB command
fix formatting issues. 

## Steps to Reproduce (for Bugs)
Check https://docs.openhab.org/addons/bindings/robonect/readme.html

## Context
Users may be confused on how to use the binding.

## Your Environment
Not relevant

**Comment by wborn** [comment]
> fix formatting issues

Looks like these formatting issues are specific to the old openHAB documentation website. It shows correctly on the [new documentation website](https://www.openhab.org/addons/bindings/robonect/). On GitHub the [README.md](https://github.com/openhab/openhab2-addons/blob/master/addons/binding/org.openhab.binding.robonect/README.md) also has no formatting issues . I don't think it's possible to create lists in markdown tables without using the HTML.

Can you create a PR to correct the remaining documentation issues @reyem?

**Comment by reyem** [comment]
@wborn yes, once I find time I will create a PR for these issues. But good to know the html issue can be ignored. thx.
